flexget/utils/requests.py

```diff
@@ -238,6 +238,11 @@ def request(self, method, url, *args, **kwargs):
         try:
             log.debug('Fetching URL %s with args %s and kwargs %s', url, args, kwargs)
             result = super(Session, self).request(method, url, *args, **kwargs)
+            try:
+                log.trace('Contents for URL %s start on next line', url)
+                log.trace('%s', result.text)
+            except UnicodeDecodeError:
```

Comment:
@tobinjt I think he just meant we are dealing with that logging crash in #1558, and we shouldn't sprinkle code that handles that exception around every logging call.